docs/typescript.mdx

```diff
@@ -2,10 +2,23 @@
 title: 'TypeScript'
 ---
 
-Emotion includes TypeScript definitions for `@emotion/react` and `@emotion/styled`. These definitions also infer types for css properties with the object syntax, HTML/SVG tag names, and prop types.
+Emotion includes TypeScript definitions for `@emotion/react` and `@emotion/styled`. These definitions infer types for css properties with the object syntax, HTML/SVG tag names, and prop types.
 
 ## @emotion/react
 
+The easiest way to use the css prop with TypeScript is with the new JSX
+transform and the `jsxImportSource` TSConfig option. For this approach, your
+TSConfig `compilerOptions` should contain
```

Comment:
```suggestion
The easiest way to use the css prop with TypeScript is with the new JSX
transform and the `jsxImportSource` TSConfig option (available since TS 4.1). For this approach, your
TSConfig `compilerOptions` should contain
```